Cql/Cql.Packaging/PublicAPI.Unshipped.txt

```diff
@@ -23,13 +23,19 @@ Hl7.Cql.Packaging.CqlTypeToFhirMapping.CqlTypeToFhirMapping(Hl7.Cql.Packaging.Cq
 Hl7.Cql.Packaging.CqlTypeToFhirMapping.CqlTypeToFhirMapping(Hl7.Fhir.Model.FHIRAllTypes? FhirType, Hl7.Cql.Primitives.CqlPrimitiveType? CqlType, Hl7.Cql.Packaging.CqlTypeToFhirMapping? ElementType = null) -> void
 Hl7.Cql.Packaging.CqlTypeToFhirMapping.Deconstruct(out Hl7.Fhir.Model.FHIRAllTypes? FhirType, out Hl7.Cql.Primitives.CqlPrimitiveType? CqlType, out Hl7.Cql.Packaging.CqlTypeToFhirMapping? ElementType) -> void
 Hl7.Cql.Packaging.PackageArgs
-Hl7.Cql.Packaging.PackageArgs.PackageArgs(System.IO.DirectoryInfo! elmDir, System.IO.DirectoryInfo! cqlDir, System.Action<System.Collections.Generic.IEnumerable<Hl7.Fhir.Model.Resource!>!>? afterPackageMutator = null, string? resourceCanonicalRootUrl = null) -> void
 Hl7.Cql.Packaging.PackageArgs.ElmDir.get -> System.IO.DirectoryInfo!
 Hl7.Cql.Packaging.PackageArgs.CqlDir.get -> System.IO.DirectoryInfo!
 Hl7.Cql.Packaging.PackageArgs.AfterPackageMutator.init -> void
 Hl7.Cql.Packaging.PackageArgs.AfterPackageMutator.get -> System.Action<System.Collections.Generic.IEnumerable<Hl7.Fhir.Model.Resource!>!>?
+Hl7.Cql.Packaging.PackageArgs.LRUCacheSize.get -> int
```

Comment:
It does not feel right to me that we have to pass in the cache size everywhere, even if the caller has no idea why. E.g. `LoadElm` and `EvaluateExpression` now take a cache size argument, but it feels too much like we're exposing an internal detail.  In other parts of the SDK I have used a static global (which is an anti-pattern by itself) for the cache size.  Any other options?

Reply:
I don't think the cache size needs to be specified during the packaging process as it's only needed during execution so it is being removed.